workshops/migrations/0012_auto_20150612_0807.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+TRANSLATE_NAMES = {
+    'Git': ['swc/git-novice'],
+    'Make': [],
+    'Matlab': ['swc/matlab-novice-inflammation'],
+    'Mercurial': ['swc/hg-novice'],
+    'Python': ['swc/python-novice-inflammation', 'dc/python-ecology'],
+    'R': ['swc/r-novice-inflammation', 'dc/r-ecology'],
+    'Regexp': [],
+    'SQL': ['swc/sql-novice-survey', 'dc/sql-ecology'],
+    'Subversion': [],
+    'Unix': ['swc/shell-novice', 'dc/shell-ecology'],
+    None: ['dc/excel-ecology']
+}
+
+EXTRA_LEGACY_NAMES = ['MATLAB']
+
+
+def add_new_lesson_names(apps, schema_editor):
+    '''Add instances of Lesson named after lessons.'''
+    Lesson = apps.get_model('workshops', 'Lesson')
+    for (old_name, new_names) in TRANSLATE_NAMES.items():
+        for name in new_names:
+            Lesson.objects.create(name=name)
```

Comment:
You don't use `old_name` here, so you should probably start the loop with:

    for new_names in TRANSLATE_NAMES.values():